test/gtest/uct/test_stats.cc

```diff
@@ -0,0 +1,385 @@
+
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2001-2014.  ALL RIGHTS RESERVED.
+* Copyright (C) UT-Battelle, LLC. 2014. ALL RIGHTS RESERVED.
+* See file LICENSE for terms.
+*/
+extern "C" {
+#include <uct/api/uct.h>
+#include <uct/tl/tl_base.h>
+
+#include <ucs/time/time.h>
+}
+#include <ucs/gtest/test.h>
+#include "uct_test.h"
+#include "uct_p2p_test.h"
+
+#if ENABLE_STATS
+
+class test_uct_stats : public uct_p2p_test {
+public:
+    test_uct_stats() : uct_p2p_test(0) {
+    }
+
+    virtual void init() {
+
+
+        ucs_stats_cleanup();
+        push_config();
+        modify_config("STATS_DEST",    "file:/dev/null");
+        //modify_config("STATS_DEST",    "stdout");
+        modify_config("STATS_TRIGGER", "exit");
+        ucs_stats_init();
+        ASSERT_TRUE(ucs_stats_is_active());
+
+        uct_p2p_test::init();
```

Comment:
i need to init stats before p2p test does its magic. Otherwise interfaces and endpoints will be created with stats disabled